HousingFinanceInterimApi/V1/UseCase/LoadSuspenseCashTransactionsUseCase.cs

```diff
@@ -0,0 +1,128 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using HousingFinanceInterimApi.V1.Domain;
+using HousingFinanceInterimApi.V1.Factories;
+using HousingFinanceInterimApi.V1.Gateways.Interface;
+using HousingFinanceInterimApi.V1.UseCase.Interfaces;
+using System.Threading.Tasks;
+using Google.Apis.Drive.v3.Data;
+using HousingFinanceInterimApi.V1.Boundary.Response;
+using HousingFinanceInterimApi.V1.Handlers;
+using HousingFinanceInterimApi.V1.Infrastructure;
+
+namespace HousingFinanceInterimApi.V1.UseCase
+{
+    public class LoadSuspenseCashTransactionsUseCase : ILoadSuspenseCashTransactionsUseCase
+    {
+        private readonly IBatchLogGateway _batchLogGateway;
+        private readonly IBatchLogErrorGateway _batchLogErrorGateway;
+        private readonly IUPCashLoadSuspenseAccountsGateway _upCashLoadSuspenseAccountsGateway;
+        private readonly IGoogleFileSettingGateway _googleFileSettingGateway;
+        private readonly IGoogleClientService _googleClientService;
+
+        private readonly string _waitDuration = Environment.GetEnvironmentVariable("WAIT_DURATION");
+
+        private const string CashSuspenseLabel = "Cash Suspense Transactions";
+
+        public LoadSuspenseCashTransactionsUseCase(
+            IBatchLogGateway batchLogGateway,
+            IBatchLogErrorGateway batchLogErrorGateway,
+            IUPCashLoadSuspenseAccountsGateway upCashLoadSuspenseAccountsGateway,
+            IGoogleFileSettingGateway googleFileSettingGateway,
+            IGoogleClientService googleClientService)
+        {
+            _batchLogGateway = batchLogGateway;
+            _batchLogErrorGateway = batchLogErrorGateway;
+            _upCashLoadSuspenseAccountsGateway = upCashLoadSuspenseAccountsGateway;
+            _googleFileSettingGateway = googleFileSettingGateway;
+            _googleClientService = googleClientService;
+        }
+
+        public async Task<StepResponse> ExecuteAsync()
+        {
+            LoggingHandler.LogInfo($"Starting suspense cash import");
+
+            const string sheetName = "Cash";
+            const string sheetRange = "A:E";
+
+            var batch = await _batchLogGateway.CreateAsync(CashSuspenseLabel).ConfigureAwait(false);
+            var googleFileSettings = await GetGoogleFileSetting(CashSuspenseLabel).ConfigureAwait(false);
+
+            if (googleFileSettings == null)
```

Comment:
Please place the object creation in multiple lines and wrap the if-statement in { } parentheses